content/yaml/runninglocally.md

```diff
@@ -29,22 +29,30 @@ To initialize keychain at system default keychain path with empty keychain passw
     keychain initialize
     keychain add-certificates --certificate /path/to/certificate.p12 --certificate-password CERTIFICATE_PASSWORD
 
+Note that `--certificate` option can be either a path literal, or a glob pattern to match certificates.
+
 To use the provisioning profile from `/path/to/profile.mobileprovision` in your Xcode project `/path/to/MyProject.xcodeproj` and generate an .ipa archive using `MyScheme` scheme, use [xcode-project](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/README.md#xcode-project) with the following command:
 
     xcode-project use-profiles --project /path/to/MyProject.xcodeproj --profile /path/to/profile.mobileprovision
     xcode-project build-ipa --project /path/to/MyProject.xcodeproj --scheme MyScheme
 
+Note that `--project` option can be either a path literal, or a glob pattern to match projects in working directory (Default is `**/*.xcodeproj`).
+
+`--profile` option can be a glob pattern as well (Default is `$HOME/Library/MobileDevice/Provisioning Profiles/*.mobileprovision`).
+
 ## Generate a universal APK with user specified keys from app bundle
 
```

Comment:
Would the following title work? (The thing is that we should follow the same logic and all other pages use .apk, not APK)
## Building universal .apk from an existing app bundle with user-specified keys